lang_js/parsing/Makefile

```diff
@@ -59,7 +67,8 @@ beforedepend:: lexer_js.ml
 OCAMLYACC=menhir --unused-tokens --explain --fixed-exception
 
 parser_js.ml parser_js.mli: parser_js.mly
-	$(OCAMLYACC) $<
+	  $(OCAMLYACC) $< 2> $(MENHIRLOG); $(SCRIPTDIR)/check_menhir_conflicts.sh $(MENHIRLOG) $(NUM_PERMITTED_CONFLICTS) $(LANG) parser_js.ml parser_js.mli
```

Comment:
I think you can use && instead of ; between the 2 commands, that way if the first command fails,
the whole line fails and make will report an error.